ques5.rb

```diff
@@ -0,0 +1,45 @@
+def valid_ip_format?(ip_address)
+  ip_format_regex = /\A(?:\d{1,3}\.){3}\d{1,3}\z/
+  if ip_address.match?(ip_format_regex)
```

Comment:
No need for if else here, the result of the condition will be returned by default